receiver/dockerstatsreceiver/testdata/mock/two_containers/stats2.json

```diff
@@ -93,7 +93,7 @@
       "usage_in_usermode": 10000000
     },
     "online_cpus": 1,
-    "system_cpu_usage": 14930240000000,
```

Comment:
I agree it's not ideal. We query the daemon for each container, and this returns assumedly a slightly varied system usage each time. The reason I had to change this value is because the results order is not deterministic, and since we take the last result. I initially made the assumption that the values returned by the daemon for system usage would have negligible difference between containers, but it seems that this isn't actually the case. 

Prompted by your comment, I ran the dockerstatsreceiver locally to see what the difference of system usage would be from two containers. Here's an example: 

```
system CPU usage (1): 1463078890000000
system CPU usage (2): 1463086880000000
```

A difference of 7,990,000,000 ns (about 8 seconds). I find this to be a very surprising result and much larger diff than what I expected. I'm not really sure how this is possible, perhaps my understanding of this behaviour is not as strong as I thought. 

As a result, I think I will close this PR and re visit this problem later. If anyone can come up with a reasonable explanation for this behaviour, we can reopen this PR and revisit the issue.